ee/packet2/include/packet2.h

```diff
@@ -0,0 +1,129 @@
+/*
+# _____     ___ ____     ___ ____
+#  ____|   |    ____|   |        | |____|
+# |     ___|   |____ ___|    ____| |    \    PS2DEV Open Source Project.
+#-----------------------------------------------------------------------
+# (c) 2020 Sandro Sobczyński <sandro.sobczynski@gmail.com>
+# Licenced under Academic Free License version 2.0
+# Review ps2sdk README & LICENSE files for further details.
+*/
+
+/** @file Successor of standard packet library. */
+
+#ifndef __PACKET2_H__
+#define __PACKET2_H__
+
+#include "./packet2_types.h"
+#include "./packet2_chain.h"
+#include "./packet2_vif.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    // ---
+    // Packet2 management
+    // ---
+
+    /** 
+	 * Allocate new packet2. 
+	 * @param qwords Maximum data size in qwords (128bit). 
+	 * @param type Memory mapping type. 
+	 * @returns Pointer to packet2 on success or NULL if memory allocation fail.
+	 */
+    packet2_t *packet2_create_normal(u16 qwords, enum Packet2Type type);
+
+    /** 
+	 * Allocate new packet2 for chain/vif purpose. 
+	 * @param qwords Maximum data size in qwords (128bit). 
+	 * @param type Memory mapping type. 
+	 * @param tte Tag transfer enable. 
+     * If >0 transfer tag is set during packet sending and 
```

Comment:
Spaces or tabs, not both. Happens a lot to me too when copy-pasting code. I'm not sure what the standard for ps2sdk is, if there even is any.

Reply:
Weird that vscode and notepad++ was not showing it to me. Fixed it in classic notepad!